lib/google/longrunning/operations_client.rb

```diff
@@ -139,14 +131,14 @@ def initialize \
           credentials ||= chan_creds
           credentials ||= updater_proc
         end
-        if app_name || app_version
-          warn "`app_name` and `app_version` are no longer being used in the request headers."
+        if service_path != SERVICE_ADDRESS || port != DEFAULT_SERVICE_PORT
+          warn "`service_path` and `port` parameters are deprecated and will be removed"
         end
 
-        credentials ||= Google::Gax::Credentials.default(scope: scopes)
+        credentials ||= Google::Auth::Credentials.default
```

Comment:
This is wrong only for LRO generation since LRO _needs_ to have scopes passed into it or else the channel will not have a valid endpoint to reach. I'll update https://github.com/googleapis/toolkit/pull/1665 to fix this.

Reply:
Updated.